### PR TITLE
30B add tags to tests for minimum regression test scope

### PR DESCRIPTION
**Pull request by philipandag**
Add tags for minimum regression and make it possible to use tags and any other options of robot from the `scripts/run.sh` wrapper

### CODE REVIEW

File: scripts/run.sh [hunk 1]
```diff
@@ -27,4 +27,4 @@ if [ "$#" -eq 0 ] || [ ! -e "$1" ]; then
   exit 1
 fi
 
-execute_robot "$1"
+execute_robot "$1" "${@:2}"
```
Comment (m-iwanicki) [inline scripts/run.sh:30]:
```suggestion
execute_robot "$@"
```

Unless I'm missing something

Comment (philipandag) [reply scripts/run.sh]: Yes, it should be equivalent
